python/pyfunc-server/docker/Dockerfile

```diff
@@ -20,7 +20,7 @@ ARG MODEL_URL
 WORKDIR /pyfunc-server
 RUN gsutil cp -r ${MODEL_URL} .
 RUN /bin/bash -c ". activate merlin-model && \
-    sed -i 's/pip$/pip=22.2.2/' model/conda.yaml && \
+    sed -i 's/pip$/pip=20.2.4/' model/conda.yaml && \
```

Comment:
Why do we use pip 20.2.4 instead of 22.2.2 here? The rest of env*.yaml files use 22.2.2 though

Reply:
This pip is used in user environment. We default to older version to allow dependency conflict as pointed out by @pradithya